extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [UNRELEASED]
 
+- Adjust test log output to display diffs only when comparing failed test results with expected test results. [#920](https://github.com/github/vscode-codeql/pull/920)
```

Comment:
Looks like you re-added this entry when you merged (it was released in 1.5.3).

Reply:
Ghaa. Copied to use as template and it looked so real, I left it in there :P Thanks for spotting